libphal/phal_sbe.C

```diff
@@ -327,37 +327,43 @@ sbeError_t capturePOZFFDC(struct pdbg_target *target)
 		std::copy(bufPtr.getData() + offset,
 			  bufPtr.getData() + offset + lenInBytes,
 			  std::back_inserter(data));
-		mulSlidData.emplace(std::make_pair(slid, data));
+		uint8_t severity = ffdc->severity;
+		mulSlidData.emplace(slid, std::make_pair(severity, data));
 		offset += lenInBytes;
 	}
 	// now combine data with same slid numbers
-	std::unordered_map<uint16_t, std::vector<uint8_t>> slidData;
+	std::map<uint8_t, std::pair<uint8_t, std::vector<uint8_t>>> slidData;
```

Comment:
Same comment as on LN314

Reply:
not sure how it got overwritten will put it back